apps/backend/src/users/pantry.entity.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  ManyToOne,
+  JoinColumn,
+} from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class Pantry {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
```

Comment:
@Column({
    type: 'varchar',
    length: 255,
  })
  
  we can specify the number of characters for the name and address columns (not hard requirement but good to have)